src/pages/IdeLoader/index.tsx

```diff
@@ -0,0 +1,324 @@
+        className = hasError ? 'error' : 'progress';
+      }
+      return (
+        <React.Fragment>
+          {this.getIcon(step, iconClass)}
+          <span className={className}>{title}</span>
+        </React.Fragment>
+      );
+    };
+
+    return [
+      {
+        id: LoadIdeSteps.INITIALIZING,
+        name: getTitle(
+          LoadIdeSteps.INITIALIZING,
+          'Initializing',
+          'wizard-icon'),
+        canJumpTo: currentStep >= LoadIdeSteps.INITIALIZING,
+      },
+      {
+        id: LoadIdeSteps.START_WORKSPACE,
+        name: getTitle(
+          LoadIdeSteps.START_WORKSPACE,
+          'Waiting for workspace to start',
+          'wizard-icon'),
+        canJumpTo: currentStep >= LoadIdeSteps.START_WORKSPACE,
+      },
+      {
+        id: LoadIdeSteps.OPEN_IDE,
+        name: getTitle(
+          LoadIdeSteps.OPEN_IDE,
+          'Open IDE',
+          'wizard-icon'),
+        canJumpTo: currentStep >= LoadIdeSteps.OPEN_IDE,
+      },
+    ];
+  }
+
+  public render(): React.ReactElement {
+    const { workspaceName, workspaceId, ideUrl, hasError, currentStep } = this.props;
+    const { alertVisible, loaderVisible } = this.state;
+
+    if (ideUrl) {
+      return (
+        <div className="ide-iframe-page">
+          {loaderVisible && (
+            <div className="main-page-loader">
+              <div className="ide-page-loader-content">
+                <img src="../../../assets/branding/loader.svg" />
+              </div>
+            </div>
+          )}
+          <iframe id="ide-iframe" src="../../../static/loader.html" />
+        </div>
+      );
+    }
+
+    return (
+      <React.Fragment>
+        {alertVisible && (
+          <AlertGroup isToast>
+            <Alert
+              variant={this.alert.variant}
+              title={this.alert.title}
+              actionClose={<AlertActionCloseButton onClose={this.hideAlert} />}
+            />
+          </AlertGroup>
+        )}
+        <Header title={`Starting workspace ${workspaceName}`}
+          status={WorkspaceStatus[hasError ? WorkspaceStatus.ERROR : WorkspaceStatus.STARTING]} />
+        <PageSection variant={SECTION_THEME} className="ide-loader-tabs">
+          <Tabs activeKey={this.state.activeTabKey} onSelect={this.handleTabClick}>
+            <Tab eventKey={IdeLoaderTabs.Progress} title={IdeLoaderTabs[IdeLoaderTabs.Progress]}>
+              {(this.state.currentRequestError) && (
+                <Alert
+                  style={{ marginTop: '15px' }}
+                  variant={AlertVariant.danger} isInline title={this.state.currentRequestError}
+                  actionClose={<AlertActionCloseButton
+                    onClose={() => this.setState({ currentRequestError: '' })} />}
+                />
+              )}
+              <Wizard
+                className="ide-loader-wizard"
+                steps={this.getSteps()}
+                ref={this.wizardRef}
+                footer={('')}
```

Comment:
Passing an empty string left footer doesn't removes the default footer.

```suggestion
                footer={(<span/>)}
```

Reply:
I have done